### PR TITLE
Clarify Shaderc readme.

### PR DESCRIPTION
**Pull request by dneto0**
Describe the relation to KhronosGroup/glslang.
We're not forking the glslang project!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,9 +28,17 @@ for more information. See also the [`AUTHORS`](AUTHORS) and
 - `third_party/`: third party open source packages; see below
 - `utils/`: utility scripts for Shaderc
 
-Shaderc depends on a [fork](https://github.com/google/glslang) of the Khronos
-reference GLSL compiler glslang. Shaderc also depends on the testing framework
-[Google Mock](https://code.google.com/p/googlemock/).
+Shaderc depends on `Glslang`, the Khronos reference compiler for GLSL.
```
Comment (antiagainst) [inline README.md]: I think glslang should be spelled as all lowercase?

**Comment by antiagainst** [comment]
Also don't forget to rebase ;)

**Comment by dneto0** [comment]
Rebased, and changed Glslang -> glslang.

**Comment by antiagainst** [comment]
LGTM

**Comment by dmircevski** [comment]
Well, [https://github.com/google/glslang](https://github.com/google/glslang) _does_ say it's "forked from KhronosGroup/glslang" right up top.  That's because we use GitHub's [fork-and-pull workflow](https://help.github.com/articles/using-pull-requests/).

DEVELOPMENT.howto.md also uses the word "fork."

**Comment by dneto0** [comment]
Yes.  But I had specific feedback that there was still confusion.  
People are busy, so it's best to write plainly.

**Pull request merged**